overlord/snapstate/handlers.go

```diff
@@ -1918,6 +1897,20 @@ func (m *SnapManager) doLinkSnap(t *state.Task, _ *tomb.Tomb) (err error) {
 	return nil
 }
 
+func setMigrationFlagsinState(snapsup *SnapSetup, snapst *SnapState) {
```

Comment:
this should have the argument swapped, our set functions tend to take the target first